src/controllers/api/reactions.js

```diff
@@ -0,0 +1,63 @@
+const { Thought } = require("../../models");
+
+const addReaction = async (req, res) => {
+  try {
+    const reactionBody = req.body;
```

Comment:
Can we validate this? If there are certain keys we are expecting from the body, I would destructure these here and then add an if condition to make sure all expected keys are present/if not all present put out your error.